### PR TITLE
All changes made to test out the robot

### PR DESCRIPTION
**Pull request by MajorDarius**
PTAL

### CODE REVIEW

File: src/Subsystems/Intake.h [hunk 1]
```diff
@@ -8,8 +8,7 @@ class Intake: public frc::Subsystem {
 private:
 	static const char kSubsystemName[];
 	static std::shared_ptr<Intake> self;
-  WPI_TalonSRX leftIntake;
-  WPI_TalonSRX rightIntake;
+  WPI_TalonSRX intakeMotor;
```
Comment (duane701) [inline src/Subsystems/Intake.h:12]: Let's see ... the name of the class is `Intake` and there is a member called `intakeMotor`. Seems a bit redundant. We still do this, but we should consider different naming where the name of the class is not part of the member variable. 

e.g. `motor`

No change necessary.

File: src/Robot.cpp [hunk 2]
```diff
@@ -39,8 +40,10 @@ std::string SideToString(robovikes::GameData::Side side){
 
 void Robot::RobotInit() {
 	RobotMap::init();
-		Chassis::getInstance();
+		Arm::getInstance();
+    Chassis::getInstance();
 		Intake::getInstance();
+		Ramp::getInstance();
```
Comment (duane701) [inline src/Robot.cpp:46]: White space seems off here. Let's change this next time.

File: src/Subsystems/Arm.cpp [hunk 3]
```diff
@@ -58,6 +59,7 @@ void Arm::SetUpTalons() {
   leftArmMotor.ConfigReverseLimitSwitchSource(LimitSwitchSource_FeedbackConnector, LimitSwitchNormal_NormallyOpen, kTimeout_10Millis);
   leftArmMotor.SetSensorPhase(true);
   leftArmMotor.SetInverted(true);
+  leftArmMotor.ConfigPeakOutputForward(0.4, kTimeout_10Millis);
```
Comment (duane701) [inline src/Subsystems/Arm.cpp]: We need to have `configPeakOutputReverse` as well. Use `-0.4` for reverse.

File: src/Commands/BrakeOff.cpp [hunk 4]
```diff
@@ -1,7 +1,16 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2017-2018 FIRST. All Rights Reserved.                        */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
```
Comment (duane701) [inline src/Commands/BrakeOff.cpp:6]: Seems odd to have this comment block here.

File: src/Commands/SetArmPosition.cpp [hunk 5]
```diff
@@ -19,7 +19,7 @@ void SetArmPosition::Execute() {
 
 // Make this return true when this Command no longer needs to run execute()
 bool SetArmPosition::IsFinished() {
- static const int kAcceptableError = 40;
+ static const int kAcceptableError = 1000;
```
Comment (duane701) [inline src/Commands/SetArmPosition.cpp:22]: todo -- Make `kAcceptableError` a percentage of the full range.

**Review by duane701 — CHANGES_REQUESTED** [review]
Add the ConfigReverse.

FTP

**Comment by MajorDarius** [comment]
PTAL

**Review by duane701 — APPROVED** [review]
LGTM

**Pull request merged**